scripts/irods/test/test_misc.py

```diff
@@ -196,3 +196,24 @@ def test_server_removes_leftover_temporary_rulebase_pid_files_on_startup__issue_
             for fn in [pid_file_1, pid_file_2, pid_file_3]:
                 self.assertFalse(os.path.exists(fn))
 
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "skip for topology testing")
+    def test_server_respawns_processes__issue_4977(self):
```

Comment:
handle the delay server.

make sure it doesn't stand up more than one delay server.

Reply:
:+1: